src/test/java/org/shikshalokam/backend/ep/TestUserRoleExtensionCreate.java

```diff
@@ -0,0 +1,96 @@
+package org.shikshalokam.backend.ep;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.shikshalokam.backend.PropertyLoader;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.net.URI;
+
+// Validating user role extension create scenarios
+public class TestUserRoleExtensionCreate extends ElevateProjectBaseTest {
+    static final Logger logger = LogManager.getLogger(TestUserRoleExtensionCreate.class);
+    private String USER_ROLE_CREATE_ENDPOINT = PropertyLoader.PROP_LIST.getProperty("elevate.createuserroleext.endpoint");
+    public static String BASE_URL = PropertyLoader.PROP_LIST.getProperty("elevate.qa.api.base.url");
```

Comment:
BASE_URL url should goto  ElevateProjectBaseTest . Please refer MentorED base Class